### ISSUE TITLE
[Feature Request] [Enhancement] 无障碍阅读支持的不全。使用微软 “讲述人” 工具，在disabled的状态讲述人无法读出组件的状态，例如el-checkbox组件，在disabled的状态不会读出是否勾选的状态。还有无法操作tabs组件，使用讲述人的时候，当第一个tabs面板里面的东西读完了，却无法通过回车切换到第二个tabs面板。

### ISSUE THREAD

**Issue by Yaojie1** [element-plus/element-plus#7347]
Feature type: **`Enhancement`**

## Details

- Enhanced compone nt name: `All`

## Description

使用微软 “讲述人” 工具，在disabled的状态讲述人无法读出组件的状态，例如el-checkbox组件，在disabled的状态不会读出是否勾选的状态。还有无法操作tabs组件，使用讲述人的时候，当第一个tabs面板里面的东西读完了，却无法通过回车切换到第二个tabs面板。

### Additional comments

使用微软 “讲述人” 工具，在disabled的状态讲述人无法读出组件的状态，例如el-checkbox组件，在disabled的状态不会读出是否勾选的状态。还有无法操作tabs组件，使用讲述人的时候，当第一个tabs面板里面的东西读完了，却无法通过回车切换到第二个tabs面板。

**Comment by Giwayume** [comment]
I'm working through accessibility problems, will likely eventually work on this.

**Comment by Giwayume** [comment]
**el-checkbox**

I can't reproduce the problem where narrator doesn't read out disabled checkbox state. Disabled checkboxes are a general usability problem with screen readers, per spec they should not be focusable. This means a screen reader user must enter "browse" or "scan" mode in order to be able to know the state of a disabled checkbox.

If you have accessibility concerns around disabled checkboxes, the general recommendation is don't use them, they're not a great experience for any screen reader.

This appears to work as expected, Microsoft Narrator reads out the state of a disabled checkbox in scan mode.

**el-tabs**

Element plus implements [Tabs with automatic activation](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html).

The enter key is not involved in selecting the tabs, the user uses the left/right arrow keys when the tab is focused to change tabs, and the tab is automatically selected. It's working to spec.

Note: you must turn **off** scan mode in Microsoft Narrator (CAPS LOCK + SPACE key) in order to interact with the web page using the standardized keyboard controls for a tab list. Otherwise you are subject to the [scan mode controls](https://support.microsoft.com/en-us/topic/7b2af804-5a2f-90fd-b0e0-672f7cbbf2da) which do not allow tab manipulation.

**Comment by JeremyWuuuuu** [comment]
Closing due to @Giwayume's comments. You should doublecheck the usage of Narrator
